libobs/obs-cocoa.m

```diff
@@ -713,14 +738,9 @@ static inline void free_hotkeys_platform(obs_hotkeys_platform_t *plat)
     if (!plat)
         return;
 
-    if (plat->monitor) {
-        [NSEvent removeMonitor:(__bridge id _Nonnull)(plat->monitor)];
-        plat->monitor = NULL;
-    }
-
-    if (plat->local_monitor) {
-        [NSEvent removeMonitor:(__bridge id _Nonnull)(plat->local_monitor)];
-        plat->local_monitor = NULL;
+    if (plat->eventTap) {
+        CGEventTapEnable(plat->eventTap, false);
+        CFRelease(plat->eventTap);
```

Comment:
You should probably also set `plat->eventTap` explicitly to `NULL` after releasing the reference to it like we do for the other struct members.

Reply:
Added.